thefuck/rules/composer_not_package.py

```diff
@@ -0,0 +1,42 @@
+import re
+from thefuck.utils import replace_argument, for_app
+
+
+@for_app('composer')
+def match(command):
+    # determine error type
+    # matching "did you mean this" is not enough as composer also gives spelling suggestions for mistakes other than mispelled commands
+    is_invalid_argument_exception = r"[InvalidArgumentException]" in command.output
+    is_unable_to_find_package = re.search(r"Could not find package (.*)\.", command.output) is not None
+    suggestions_present = (('did you mean this?' in command.output.lower()
+                            or 'did you mean one of these?' in command.output.lower()))
+    return is_invalid_argument_exception and is_unable_to_find_package and suggestions_present
```

Comment:
i switched to black and now flake8 complains about E203 because I made a function call within list slicing. it appears to be acknowledge by creators of black and they suggest ignoring it. see below.